css/style.css

```diff
@@ -359,82 +613,386 @@
+  }
+
+  .safe-list-item__text {
+    margin: 0 50px 0 0;
+    padding-top: 5px;
+    min-width: 100px;
+  }
+
+  .safe-list-item__star {
+    margin-right: 5px;
+  }
+
+  .model-img {
+    position: absolute;
+    width: 80%;
+    height: auto;
+    top: 80px;
+    left: 90px;
+  }
+
+  .model-img-car {
+    display: none;
+  }
+
+  .model-img-car-ipad {
+    order: 0;
+    padding-left: 0;
+    padding-top: 80px;
+    padding-bottom: 50px;
+    width: 95%;
+    height: 100%;
+  }
+
+  .container-safe-star {
+    display: none;
+    margin-bottom: 50px;
+    background-image: url(/img/security.svg);
+    background-repeat: no-repeat;
+    background-position-y: 10px;
+  }
+
+  .container-safe-text {
+    width: 380px;
+    padding-left: 35px;
+    margin-right: 40px;
+  }
+
+  .safe-p {
+    margin: 0;
+    padding: 0;
+  }
+
+  .container-safe-list__title {
+    margin-bottom: 24px;
+    font-size: 30px;
+    line-height: 37px;
+  }
+
+  .container-safe-star__text {
+    margin: 0;
+    padding-bottom: 0;
+    padding-left: 80px;
+    font-size: 50px;
+    line-height: 61px;
+    font-weight: bold;
+  }
+
+  .container-safe-star__description {
+    margin: 0;
+    padding-left: 80px;
+    font-size: 25px;
+    line-height: 30px;
+    font-weight: medium;
+  }
+
+  .article-footer-text {
+    margin-right: 10px;
+    padding-top: 7px;
+    text-transform: uppercase;
+  }
+
+  .footer-link {
+    display: none;
+    margin-left: auto;
+    align-self: flex-end;
+  }
+}
+
+@media (min-width: 769px) {
+  html,
+  body {
+    margin: 0;
+    font-family: 'montserrat-font', sans-serif;
+    font-size: 15px;
+    font-weight: normal;
+    color: #000;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  .btn {
+    padding: 10px 45px;
+    font-size: 13px;
+    line-height: 13px;
+    font-weight: bold;
+    text-transform: uppercase;
+    background-color: #fff;
+    border: 2px solid #e51937;
+    cursor: pointer;
+    transition: 1s;
+  }
+
+  .btn:hover {
+    transform: scale(1.1);
+  }
+
+  .header {
+    margin-bottom: 60px;
+  }
+
+  .container {
+    margin: 0 auto;
+    width: 1440px;
+  }
+
+  .container-header {
+    display: flex;
+    align-items: center;
+    padding-top: 25px;
+    padding-left: 31px;
+  }
+
+  .link-logo {
+    padding-left: 10px;
+  }
+
+  .menu {
+    margin-left: auto;
+    margin-right: 60px;
+    margin-bottom: 5px;
+  }
+
+  .menu__link {
+    position: relative;
+    margin-right: 30px;
+    line-height: 18px;
+    color: #000;
+    transition: color 1s ease-in-out;
+  }
+
+  .menu__link:nth-child(1) {
+    margin-right: 60px;
+    color: #e51937;
+  }
+
+  .menu__link:nth-child(3) {
+    margin-right: 40px;
+  }
+
+  .menu__link:nth-child(4) {
+    margin-right: 45px;
+  }
+
+  .menu__link:nth-child(5) {
+    margin-right: 221px;
+  }
+
+  .menu__link:nth-child(6) {
+    margin-right: 37px;
+  }
+
+  .menu__link:hover {
+    color: #e51937;
+  }
+
+  .menu__link:last-child {
+    margin-right: 0;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6147400)